src/stratigraphy/util/find_description.py

```diff
@@ -102,3 +165,35 @@ def _block_separated_by_line(
         if is_line_long_enough and line_ends_block:
             return True
     return False
+
+
+def _block_separated_by_lefthandside_line_segment(
```

Comment:
I'd like to think here on how to split blocks as well where the lefthandside line does not cross the block. 

Example here:
![image](https://github.com/swisstopo/swissgeol-boreholes-dataextraction/assets/43504052/3d097cda-5796-43e4-98f4-9268ccc178d3)